core/controllers/collection_editor_test.py

```diff
@@ -218,6 +218,39 @@ def test_editable_collection_handler_put_can_access(self):
 
         self.logout()
 
+    def test_cannot_put_long_commit_message(self):
+        """Check that putting a long commit message is denied."""
```

Comment:
This test seems way more complex than necessary to test what you are testing. Please reduce it to the simplest possible test.

Ditto elsewhere. Do not simply copy the previous test.

Reply:
Done